package.json

```diff
@@ -35,7 +35,7 @@
     "cluster:start": "./aio/scripts/start-cluster.sh",
     "cluster:stop": "./aio/scripts/stop-cluster.sh",
     "check": "concurrently \"npm run check:backend\" \"npm run check:frontend\" \"npm run check:license\"",
-    "check:backend": "./aio/scripts/lint-backend.sh",
+    "check:backend": "cd ${GOPATH}/src/github.com/kubernetes/dashboard && golangci-lint run -c .golangci.yml src/app/backend/...",
```

Comment:
We cannot cd here.

Reply:
(Theoretically) You should already be in the root Dashboard folder when running any commands. This holds true within Travis as well.

Reply:
Yes we can! Current directory after `cd` is under `.tmp`. https://travis-ci.org/kubernetes/dashboard/jobs/508186409#L331

Reply:
But it is not a good practise, to change directory from a script like this.

Reply:
Extremely agree. But I could not find right way for now. I'd like to investigate more for this problem in subsequent PR.

Reply:
It is not resolved.

Reply:
This issue is investigated in #3677 and #3678 individually from this PR.

Reply:
Yep, but we cannot merge it with `cd` here.

Reply:
OK. Let's proceed #3677 and #3679 before this. But it's time to finish today's work. Since tomorrow is national holiday, so I will tackle them again at Friday.

Reply:
Sure, thank you for your effort in fixing this. Really appreciate that! Have a good holiday :)